### PR TITLE
RHBA-365 - Pagination displays incorrect amount of tasks when custom …

### PR DESCRIPTION
**Pull request by mswiderski**
…advanced query returns duplicate records

### CODE REVIEW

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/preprocessor/BusinessAdminTasksPreprocessor.java [hunk 1]
```diff
@@ -60,7 +68,18 @@ public void preprocess(DataSetLookup lookup) {
             filter.addFilterColumn(columnFilter);
             lookup.addOperation(filter);
         }
-
+        if (lookup.getLastGroupOp() == null) {
```
Comment (tsurdilo) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/preprocessor/BusinessAdminTasksPreprocessor.java]: Could be extracted in util method maybe cause seems to be reusable logic?

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/preprocessor/PotOwnerTasksPreprocessor.java [hunk 2]
```diff
@@ -69,7 +77,19 @@ public void preprocess(DataSetLookup lookup) {
             filter.addFilterColumn(columnFilter);
             lookup.addOperation(filter);
         }
-
+        
+        if (lookup.getLastGroupOp() == null) {
```
Comment (tsurdilo) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/preprocessor/PotOwnerTasksPreprocessor.java]: same here

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/preprocessor/BusinessAdminPreprocessor.java [hunk 3]
```diff
@@ -65,5 +73,17 @@ public void preprocess(DataSetLookup lookup) {
             filter.addFilterColumn(columnFilter);
             lookup.addOperation(filter);
         }
+        if (lookup.getLastGroupOp() == null) {
```
Comment (tsurdilo) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/preprocessor/BusinessAdminPreprocessor.java]: same here

**Comment by mswiderski** [comment]
Jenkins execute full downstream build

**Comment by mswiderski** [comment]
@tsurdilo fixed the duplication

**Comment by mswiderski** [comment]
Jenkins execute full downstream build

**Pull request merged**